docs/resources/add-ins-differences.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: 'Differences between Office Scripts and Office Add-ins'
 description: 'The behavior and API differences between Office Scripts and Office Add-ins.'
-ms.date: 04/24/2020
+ms.date: 06/01/2020
 localization_priority: Normal
 ---
 
 # Differences between Office Scripts and Office Add-ins
 
-Office Add-ins and Office Scripts have a lot in common. They both offer automated control of an Excel workbook through the `Excel` namespace of the Office JavaScript API. However, Office Scripts are more limited in their scope.
+Office Add-ins and Office Scripts have a lot in common. They both offer automated control of an Excel workbook a JavaScript API. However, Office Scripts use a specialized, synchronous version of the Office JavaScript API.
```

Comment:
Maybe this is cleared up elsewhere in the article, but I was confused by the fact that this sentence seems to say that the Office Scripts APIs are synchronous but line 28 below refers to "Office Scripts Async APIs".

Reply:
There's two API models: standard and async. The standard (sync) model is just "Office Scripts API", whereas the other one is "Office Scripts Async API". The async model is not recommended for most users and needs special `Excel.run` code blocks to run. However, part of the reason for including that model was to better allow add-in code to run in Office Scripts. 

I pushed a new change to try to clarify this. Please let me know if there needs to be more.